### DISCUSSION TITLE
TUI Wishlist

### DISCUSSION THREAD

**Discussion by orhun** [ratatui/ratatui#1570] [Ideas]
Feel free to drop your wishes for applications or libraries you'd love to see built below!

Maybe someone will build it, you never know!

**Comment by orhun** [comment]
Some ideas from [Discord](https://discord.com/channels/1070692720437383208/1070692720437383211/1316508873271672932):

- TUI for setting default applications
- TUI app to do video calls
- TUI app for youtube livestream chat, similar to [twitch-tui](https://github.com/Xithrius/twitch-tui)
- TUI app for dungeon&dragons games, featuring, well, the dungeon, plus separate private channels from DM to players, dice rolling, etc. Could even be using sixel to display nice monster images

**Comment by mockersf** [comment]
a tui for obsidian
a tui for todoist
a tui for obsidian and todoist so I can use both at once

**Comment by thePanz** [comment]
A TUI to manage ZFS, bonus points if can be usd as a TUI version of TrueNAS

**Comment by ricott1** [comment]
A TUI similar to https://github.com/d99kris/nchat but also integrating signal

**Comment by josueBarretogit** [comment]
grep-tui, search all the tuis in existence

**Comment by Necoo33** [comment]
It'd be awesome if there is a PuTTY replacement empowered with ratatui. It'll be enough if connects server via telnet and we saw more nicer emulator. And it should support mouse events, for example, if we are in this route: "/home/necoo33/folder1/folder2" and if we click on the "necoo33" text we simply go to the "/home/necoo33" folder.

**Comment by danipardo** [comment]
a frontend for journalctl would be great. Filtering by unit, date range, moving up and down the logs.. because I will *never* remember all those parameters journalctl accepts.

**Reply by kemitix** [reply]
Something like lazyjournal?: https://terminaltrove.com/lazyjournal/

**Reply by danipardo** [reply]
Yes, something like this. I didn´t know of lazyjournal, but afaict it doesn´t allow filtering by date range. But yes, something like this.

**Comment by pcgeek86** [comment]
Add / remove security cameras from my Dockerized video ingestion system. Currently just scripting it out, rawdog approach.

**Comment by bartosz-skejcik** [comment]
Two players join a lobby and select their preferred programming language for solving tasks. They start at the same time and are given a set amount of time to solve algorithmic challenges. Both players can see each other's scores in real-time as they progress. Points are awarded for correct solutions, and the player who reaches the target score first or has the most points when time runs out is declared the winner.

**Comment by andrei-ng** [comment]
A Map TUI similar to https://github.com/rastapasta/mapscii would be awesome

**Comment by orhun** [comment]
We need a TUI for https://textmode.art !

**Comment by phanirithvij** [comment]
A tui for msft todos (Ideally nothing to do with msft but I have all my todos there :cry:)
